cli/tests/e2e/targets/dependency_aware/pipfile_with_empty_dev-packages/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+boto3 = "*"
+requests = "*"
+pyhcl = "*"
+
+[dev-packages]
+
+[requires]
```

Comment:
Not sure if it would even be a valid Pipfile, so I'll ask: should we / would these changes handle two section headers without a newline between them?

```
[dev-packages]
[requires]
```

Reply:
good callout, nothing in the spec makes your case invalid. I'll add a test and see what happens :)

Reply:
I added a test case and it passes!